bin/docker-build-debug

```diff
@@ -3,22 +3,20 @@
 set -eu
 
 if [ $# -ne 0 ]; then
-    echo "no arguments allowed for $(basename $0), given: $@" >&2
+    echo "no arguments allowed for ${0##*/}, given: $@" >&2
     exit 64
 fi
 
-bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-rootdir="$( cd $bindir/.. && pwd )"
+bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
+rootdir=$( cd "$bindir"/.. && pwd )
 
-. $bindir/_docker.sh
-. $bindir/_tag.sh
+. "$bindir"/_docker.sh
+. "$bindir"/_tag.sh
 
-dockerfile=$rootdir/Dockerfile-debug
+dockerfile="$rootdir"/Dockerfile-debug
 
 (
     "$bindir"/docker-build-base
 ) >/dev/null
 
-tag="$(head_root_tag)"
-
-docker_build debug $tag $dockerfile
+docker_build debug $(head_root_tag) "$dockerfile"
```

Comment:
why remove the quotes from head_root_tag?

Reply:
Here the behavior is the very same as it was. At assignment, the double quotes are redundant since
test=$(echo foo bar)
and
test="$(echo foo bar)"
will both end up with test having the value 'foo bar'.
And since tag was used only in one place, I found it unnecessary to have it in a separate variable and replace the unquoted $tag on the docker_build line with $(head_root_tag). So it does exactly the same thing as before, but with fewer lines and characters.
Surely, that leaves the question on if the tag should be quoted or not, and I believe the tag is not allowed to contain any characters that would interfere with the shell unquoted anyway?